js/changeColor.js

```diff
@@ -0,0 +1,63 @@
+import {storage} from './main.js'
+
+let btn = document.getElementById('change-color-btn');
+
+/* Need this since the color picker wants hex 
```

Comment:
the function name is quite self explanatory,

and explain why it is needed in more general terms, "JS converts RGB to automatically, this function ensures it stays in HEX"

Something along those lines if I understand it correctly!

Reply:
It's just an incompatibility, the color picker use hex and JS use rgb when using the style function, even if I put in a hex value

Reply:
I just remove the comment since as you said, it's pretty self explanatory